actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -251,11 +249,7 @@ def image_tag(source, options = {})
         check_for_image_tag_errors(options)
         skip_pipeline = options.delete(:skip_pipeline)
 
-        src = options[:src] = resolve_image_source(source, skip_pipeline)
-
-        unless src.start_with?("cid:") || src.start_with?("data:") || src.blank?
-          options[:alt] = options.fetch(:alt) { image_alt(src) }
-        end
+        options[:src] = path_to_image(source, skip_pipeline: skip_pipeline)
```

Comment:
Why are we changing the method being used to generate `src`? Using `path_to_image` instead of `resolve_image_source` is causing CI to fail 😬